api/actions/serializers.py

```diff
@@ -187,3 +190,6 @@ class Meta:
         related_view='requests:node-request-detail',
         related_view_kwargs={'request_id': '<target._id>'},
     )
+
+    permissions = ser.ChoiceField(choices=permissions.PERMISSIONS, required=False)
```

Comment:
I didn't include a default value in the serializer because the [action model has a default](https://github.com/CenterForOpenScience/osf.io/pull/8192/files#diff-4d8932eca0ab3ad463050ca1dcf2a3c2R40), and a default is also set in the `save_changes` method [of the request machine](https://github.com/CenterForOpenScience/osf.io/pull/8192/files#diff-f91eedaa97bf98cd4055cbac83445c66R155), so since it's not required, permissions are set elsewhere!